Tools/NTupleReader.h

```diff
@@ -357,6 +408,60 @@ class NTupleReader
         }
     }
 
+    template<typename T> const std::vector<TLorentzVector>& getVec_LVFromPtEtaPhiM(const std::string& ptVar, const std::string& etaVar, const std::string& phiVar, const std::string& massVar) const
```

Comment:
I really like this function. I feel like all functions in anyone's code should be written in this way.